clients/client-py/taskcluster/helper.py

```diff
@@ -128,3 +143,45 @@ def load_secrets(
             raise Exception("Missing value {} in secrets.".format(required_secret))
 
     return secrets
+
+
+def upload_artifact(queue_service, artifact_path, content, content_type, ttl):
+    """
+    Create an artifact on the current Taskcluster Task in 2 steps:
+    1. create the Artifact through the API
+    2. upload the file on the storage provider
+    """
+    task_id = os.environ.get("TASK_ID")
+    run_id = os.environ.get("RUN_ID")
+    assert task_id and run_id, "Can only run in Taskcluster tasks"
```

Comment:
It might be worthwhile to also check for TASKCLUSTER_PROXY_URL, as this won't work without that running.

Reply:
Done.